docs/conf.py

```diff
@@ -163,6 +163,7 @@ def role(role, rawtext, text, lineno, inliner,
         return [node], []
     return role
 
+behaviors_role = explorer_role('Behaviors ', 'skills/behaviors/%d')
```

Comment:
```suggestion
behaviors_role = explorer_role('Behavior ', 'skills/behaviors/%d')
```
Didn't notice that with the other additions, but this should be singular to produce sth like `Behavior 2345` as a link text

Reply:
Do you want me to make that change to the other roles that are plural and change them all to be singular?

Reply:
Sure, no need for separate PRs